### PR TITLE
extract files from zip in python

### PR DESCRIPTION
**Pull request by Sapna2001**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #347 

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

File: Python/Extract_Contents_of_Zip/extractScript.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#import pyshorteners library
+import pyshorteners
+
```
Comment (tejan-singh) [inline Python/Extract_Contents_of_Zip/extractScript.py]: you need to remove this URL shortener script from this pr.

File: Python/Extract_Contents_of_Zip/extractScript.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#import pyshorteners library
```
Comment (SANKET7738) [inline Python/Extract_Contents_of_Zip/extractScript.py]: Can you explain to me the use of this file here? @Sapna2001

File: Python/Extract_Contents_of_Zip/script.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from zipfile import ZipFile
+
+def main():
+    # Enter the name of zip file
+    zipFile = input("Enter zip file:")
+    # Enter the path of the file to be extracted
+    extractFile = input("Enter path of the file:")
```
Comment (tejan-singh) [inline Python/Extract_Contents_of_Zip/script.py]: you can take any one input from user. Either file name or path. are both really required?

**Review by tejan-singh — CHANGES_REQUESTED** [review]

File: Python/Extract_Contents_of_Zip/extractScript.py (hunk 2 above)
Comment (Sapna2001) [reply Python/Extract_Contents_of_Zip/extractScript.py]: I extracted this script from a zip file to show an example. I will remove it.

File: Python/Extract_Contents_of_Zip/extractScript.py (hunk 1 above)
Comment (Sapna2001) [reply Python/Extract_Contents_of_Zip/extractScript.py]: Ok

File: Python/Extract_Contents_of_Zip/script.py (hunk 3 above)
Comment (Sapna2001) [reply Python/Extract_Contents_of_Zip/script.py]: I have made the changes.

**Pull request merged**